extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -268,6 +268,7 @@ export class OpenPGPKey {
       const message = await opgp.createMessage({ text });
       return await opgp.sign({ message, format: 'armored', signingKeys: [signingPrv], detached });
     }
+    text = text ? text : '\n';
```

Comment:
Original issue happens because empty string ` ` is equal to `false` in js.
But if we'll create empty string as `String(text)` then it'll be `true`.
So let's just convert original `text` parameter to string (as `String(data)`) at https://github.com/FlowCrypt/flowcrypt-browser/blob/75621567b2ef423461d5363776e556921ff4c86a/extension/js/common/core/crypto/pgp/msg-util.ts#L162

Reply:
Hello @sosnovsky -  That would work too because if `text` is still doesn't hold any string value -- it will still failed to pass the truthy value test check - https://github.com/openpgpjs/openpgpjs/blob/026b348cf87988cf95896dd9c5dd1a0c8129ddb1/src/cleartext.js#L210.

So I think a way to solve this is to put something "invisible" in replacement when the text is completely blank to pass the checks.

Reply:
Yeah, you're right - I initially tested `new String(text)` with raw openpgp.js, but it doesn't work when used in FlowCrypt code. So let's use your solution ✅

Reply:
Thanks - Roma, I understand your side now. The way you test it out is also my go to too --- just poping my test enviroment with raw openpgp.js. Thanks for considering the suggested solution.